pom.xml

```diff
@@ -191,6 +191,29 @@
         <testJvm />
       </properties>
     </profile>
+    <profile>
+      <id>java24</id>
+      <activation>
+        <jdk>24</jdk>
+      </activation>
+      <properties>
+        <argLine.java9.extras />
+        <!-- Export some stuff which is used during our tests -->
+        <argLine.java9>--illegal-access=deny ${argLine.java9.extras}</argLine.java9>
```

Comment:
```
OpenJDK 64-Bit Server VM warning: Ignoring option --illegal-access=deny; support was removed in 17.0
```

We don't need to carry this option forward any more.

Reply:
I guess we should remove it from other profiles as well